tests/common/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 add_testcase(NAME common
     LIBS s25Common testHelpers
+    COST 1
```

Comment:
why not use labels for the tests?

https://stackoverflow.com/questions/24495412/ctest-using-labels-for-different-tests-ctesttestfile-cmake

Reply:
I thought about that and discarded it. It doesn't seem to play nice with with "make test" and I have found other reports of people wanting something like this and this was the best solution
As mentioned in the description: this test is still build. You can run it manually in debug e.g. Through VS. No problem there so I kept this